devops/scripts/benchmarks/benches/compute.py

```diff
@@ -224,6 +224,71 @@ def parse_unit_type(compute_unit):
 
 
 class ComputeBenchmark(Benchmark):
+
+    # list of benchmarks to exclude from unitrace due to SIGSEGV, SIGABRT or timeouts
+    unitrace_exclusion_list = [
```

Comment:
We should just let things fail. If unitrace is failing, it's an urgent bug that needs to be fixed.